### PR TITLE
Refactoring the Scratch card

### PR DESCRIPTION
**Pull request by mianjoto**
I redesigned the "card" system for Scratches to better fit our brand's identity.

A scratch now looks like this:
image

Before, they looked like this:
image

This is an example of what the cards now look like in the `discover` page.
image

Before, the `discover` page looked like:
image

Changelog:
- Complete redesign of the Scratch card
- Separated each Scratch into a normal and large card, which can be included in any HTML file by using the following Jinja2 statements:
    - `{% include 'scratch-card.html' %}` for a normal-sized card
    - `{% include 'scratch-card-large.html' %}` for a large card
- The following pages have been redesigned to allow for 3 scratches to display horizontally; previously 4 per row:
    - `user.html`
    - `discover.html`
    - `view-scratch.html`
- Added a `get_all_scratches_and_their_authors` method that returns a Python dictionary where each item contains a key-value pair of a Scratch object and an AppUser object that holds the data of its author.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -289,3 +311,9 @@ def page_not_found(error):
 def logout():
     session.pop('user')
     return redirect('/signin')
+
+
+# TODO: Remove this temporary test function
+@app.route('/testscratchcard')
+def testscratchcard():
+    return render_template('scratch-card.html')
```
Comment (CesarCGuzman) [inline app.py]: Remove if needed

**Review by CesarCGuzman — APPROVED** [review]
Looks good

**Review by BrianSamuels — APPROVED** [review]
Looks good, great job

**Comment by vyarabadi** [comment]
good job!! it looks good

**Review by harshicabalu — APPROVED** [review]
I am a little confused but I think I understand it and I will ask questions if I have any tomorrow

File: app.py (hunk 1 above)
Comment (mianjoto) [reply app.py]: Thanks for catching that! I'll remove it.

**Pull request merged**